app/src/main/java/com/marvelapp/presentation/activity/CharacterActivity.kt

```diff
@@ -0,0 +1,22 @@
+package com.marvelapp.presentation.activity
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.marvelapp.databinding.ActivityCharacterBinding
+
+class CharacterActivity : AppCompatActivity() {
```

Comment:
as this is not necessary in this PR, I recommend to remove it

Reply:
Solved